app/views/external_users/claims/misc_fees/_misc_fee_fields.html.haml

```diff
@@ -10,29 +10,55 @@
       %label.form-label-bold
         = t('.fee_type')
       %a{id: "misc_fee_#{@misc_fee_count}_fee_type"}
-      = f.collection_select :fee_type_id, @claim.eligible_misc_fee_types, :id, :description, {include_blank: ''.html_safe }, {class: 'form-control js-misc-fee-type js-fee-type typeahead', style: 'width:100%'}
+      = f.collection_select :fee_type_id,
+       @claim.eligible_misc_fee_types, :id, :description,
+       {include_blank: ''.html_safe },
+       {class: 'form-control js-misc-fee-type js-fee-type js-fee-calculator-fee-type typeahead', style: 'width:100%'}
 
       = validation_error_message(@error_presenter, "misc_fee_#{@misc_fee_count}_fee_type")
 
   = f.adp_text_field :quantity,
                           label: t('.quantity'),
-                          input_classes:'quantity fee-quantity form-control-1-8',
+                          hint_text: t('.quantity_hint'),
+                          hide_hint: true,
+                          input_classes:'quantity fee-quantity form-control-1-8 js-fee-quantity js-fee-calculator-quantity',
                           input_type: 'number',
                           value: fee.quantity,
                           errors: @error_presenter
 
+  .js-fee-calculator-effectee
+    .calculated-fee
+    = f.adp_text_field :rate,
+        label: t('.rate'),
+        input_classes:'rate fee-rate js-fee-calculator-rate form-input-denote__input form-control-1-4',
+        input_type: 'currency',
+        errors: @error_presenter,
+        input_readonly: f.object.rate_calculated?,
+        value: number_with_precision(f.object.rate, precision: 2)
+
+  .js-fee-calculator-success
+    = f.hidden_field :rate_calculated, value: f.object.rate_calculated?
+
+  .help-wrapper.form-group{ style: 'display: none;' }
```

Comment:
**[not a blocker]** [Ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2491/files#r221971497)